cmd/vmlabel.go

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package cmd
+
+import (
+	"github.com/kubearmor/kubearmor-client/vm"
+	"github.com/spf13/cobra"
+)
+
+var (
+	labelOptions vm.LabelOptions
+	label        string
+)
+
+// vmLabelCmd represents the vm command for label management
+var vmLabelCmd = &cobra.Command{
+	Use:   "label",
+	Short: "label handling for vm/nonk8s control plane",
+	Long:  `label handling for vm/nonk8s control plane`,
+}
+
+// vmLabelAddCmd represents the vm add label command for label management
+var vmLabelAddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "add label for vm k8s/nonk8s control plane",
+	Long:  `add label for vm k8s/nonk8s control plane`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// Create http address
+		httpAddress := "http://" + HttpIP + ":" + HttpPort
+		isNonK8sEnv := IsNonK8sEnv
+
+		if err := vm.LabelHandling("ADD", labelOptions, httpAddress, isNonK8sEnv); err != nil {
+			return err
+		}
+		return nil
+	},
+}
+
+// vmLabelDeleteCmd represents the vm add label command for label management
+var vmLabelDeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "delete label for vm k8s/nonk8s control plane",
+	Long:  `delete label for vm k8s/nonk8s control plane`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// Create http address
+		httpAddress := "http://" + HttpIP + ":" + HttpPort
```

Comment:
```suggestion
	 We can use `net.JoinHostPort` instead of direct concatenation.
```
Ref: https://pkg.go.dev/net#JoinHostPort

Reply:
@h3llix I already tried with net.JoinHostPort, but not getting an expected output. Will try and share the results.